docs/users_guide/admin_index.md

```diff
@@ -514,56 +514,109 @@ An admin user can view their profile as if they were an ordinary user on [this d
-**Sent to:**  Single new instructor  
-**Sent from:**  Regional coordinator email  
-**Subject:** Confirmation of your participation as an instructor for {% if workshop_main_type %}{{ workshop_main_type }}{% endif %} workshop organized by {{ host.fullname }} {% if dates %} ({{ dates }}){% endif %}  
-**Date sent:** One hour after conditions are met.  
-**Conditions:** New role of instructor added. Start date in future.  Tag is not stalled, canceled, or unresponsive. Has administrator.  Centrally organized only.  
-
-**Email description:** A new event is created from Self-Organised Request Form.  
-**Sent to:**  Form submitter and additional contacts  
-**Sent from:**  Regional coordinator email  
-**Subject:**  {{ workshop.host.fullname }} ({{ workshop.slug }}) Workshop  
-**Date sent:** One hour after conditions are met.  
-**Conditions:**  Start date in future.  Tag is not stalled, canceled, or unresponsive.  Event created from SO form.  Self organized only.  
-
-**Email description:** 7 days past the end date of an active workshop.  
-**Sent to:**  All Workshop Hosts + Instructors  
-**Sent from:**  Regional coordinator email  
-**Subject:** Completed {% if workshop_main_type %}{{ workshop_main_type }}{% else %}Carpentries{% endif %} workshop at {{ workshop.venue }} on {{ dates }} ({{ workshop.slug }})  
-**Date sent:** 7 days after end date.  
-**Conditions:** End date in past.  Tag is not stalled, canceled, or unresponsive. Tag is one of ["LC", "DC", "SWC", "Circuits"].  Centrally organized or Self organized.  
+AMY sends automated emails for membership, instructor training, and workshop administration. This system was completely revamped in October 2024.
+
+[Email templates](https://amy.carpentries.org/emails/templates/) and [status of scheduled emails](https://amy.carpentries.org/emails/scheduled_emails/) can be viewed directly through the user interface under the "More" menu.  Not all templated emails are currently active.
+
+Email recipients, schedule, and conditions are all set in the AMY codebase.
+
+From, Reply to, CC, and BCC are all set in the AMY interface.  The email subject and body are also set in the AMY interface. These fields can all be modified by admin users at any time.
+
+#### Active emails (Workshops)
+
+??? abstract "Admin Signs Instructor Up for Workshop"
+    **[View in AMY](https://amy.carpentries.org/emails/template/53209bdb-fbd2-4620-988b-16703d38a083/)**  
+    **Email description:**  Send email to Instructor informing them the admin has signed them up as interested in teaching a workshop  
+    **Sent to:**  Instructor   
+    **Date sent:**   One hour after admin action  
+
+??? abstract "Ask for website"
+    **[View in AMY](https://amy.carpentries.org/emails/template/8b29643c-d4f2-4b58-bc3c-5ebc4ff3d7b4/)**  
+    **Email description:**  Send email to workshop Hosts & Instructors reminding them to send us workshop website link  
+    **Sent to:**  Instructors and Hosts   
```

Comment:
This is currently coded so that only instructors are the recipients. Let me know if you want this changed.